### ISSUE TITLE
Is there a mechanism that zabbix will prompt when the monitoring item does not get the data?

### ISSUE THREAD

**Issue by admin1518** [jjmartres/Zabbix#147]
Is there a mechanism that zabbix will prompt when the monitoring item does not get the data?

**Comment by Atem18** [comment]
@admin1518 Yes, you have to create a trigger with the function nodata : https://www.zabbix.com/documentation/4.2/manual/appendix/triggers/functions

As an example, you can use the one from the template of the Zabbix agent.

Or if you want, someone made an article about it : https://werner-dijkerman.nl/2015/03/31/zabbix-nodata-trigger-really-a-lifesaver/
